ovs/extensions/services/service.py

```diff
@@ -51,7 +57,21 @@ def __getattr__(cls, item):
                         else:
                             raise RuntimeError('The ServiceManager is unrecognizable')
                     elif 'systemd' in init_info:
-                        ServiceManager.ImplementationClass = Systemd
```

Comment:
Leave this line as default for systemd
then check if fleet is installed and correctly setup --> overwrite default with FleetCtl
Then you can remove all the ImplementationClass changes in all the else stacks and except stacks